scripts/build_components.sh

```diff
@@ -65,13 +67,16 @@ for dir in "${components_to_build[@]}"; do
   full_image_names=()
   echo "Tagging image with following tags:"
   for tag in "${tags[@]}"; do
-    full_image_name=ghcr.io/${namespace}/${BASENAME}:${tag}
+    full_image_name=${namespace}/${BASENAME}:${tag}
+    if [ -n "${registry}" ] ; then
+      full_image_name=${registry}/${full_image_name}
+    fi
     echo "$full_image_name"
     full_image_names+=("$full_image_name")
   done
 
   # Prevent this from mistakenly being used below
-  unset full_image_name
+#  unset full_image_name
```

Comment:
Please double check that images are built under the right name, I remember we had this issue before

Reply:
Yes it's correct. I'll remove this instead of commenting it out though.